discojs/src/dataset/data_loader/tabular_loader.ts

```diff
@@ -75,15 +75,20 @@ export abstract class TabularLoader<Source> extends DataLoader<Source> {
     * Creates the CSV datasets based off the given sources, then fuses them into a single CSV
     * dataset.
     */
-  async loadAll (sources: Source[], config: DataConfig): Promise<Data> {
+  async loadAll (sources: Source[], config: DataConfig): Promise<DataTuple> {
     const datasets = await Promise.all(sources.map(async (source) =>
       await this.load(source, { ...config, shuffle: false })))
     const dataset = List(datasets).reduce((acc: Dataset, dataset) => acc.concatenate(dataset))
-    return {
+    const data = {
       dataset: config?.shuffle ? dataset.shuffle(BUFFER_SIZE) : dataset,
       // dataset.size does not work for csv datasets
       // https://github.com/tensorflow/tfjs/issues/5845
       size: 0
     }
+    // TODO: Implement validation split for tabular data (tricky due to streaming)
+    return {
+      train: data,
+      validation: undefined
```

Comment:
better to simply omit the field, it defaults to `undefined` :) literally not defining it haha (this is what optional params and undefined mean)